src/legacy/core_plugins/visualizations/public/index.scss

```
@@ -1,3 +1,4 @@
 @import 'src/legacy/ui/public/styles/styling_constants';
 
 @import './embeddable/_index';
+@import './np_ready/public/wizard/_index';
```

Really the `np_ready` folder should have it's own `_index.scss` file as well, but this is ok since I know there's still a lot of file shuffling happening.

Side note, with SCSS files, you don't need to include the underscore `_` in the import file names. Ex:
```suggestion
@import './np_ready/public/wizard/index';
```

---

Thanks, improved